tests/DanBettles/Telex/CandidateTest.php

```diff
@@ -24,7 +25,7 @@ public function testGetsourceReturnsTheSourceSetUsingSetsource()
         $something = $candidate->setSource('foo');
 
         $this->assertSame('foo', $candidate->getSource());
-        $this->assertSame($candidate, $something);
```

Comment:
Please restore this line.  I used `assertSame()` -- instead of just `assertInstanceOf()` -- because I wanted to ensure that `$something` is the exact same instance, not only the same type.  Thanks